internal/defence/limiter_test.go

```diff
@@ -0,0 +1,161 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package defence
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"storj.io/storj/internal/testcontext"
+)
+
+const (
+	key1           = "email1@example.com"
```

Comment:
You forgot (commented by Egon) to update these emails to use a `.test` domains
https://tools.ietf.org/html/rfc2606#page-2

Reply:
This has been missed